### DISCUSSION TITLE
dfhack-config folder: save to keep after an upgrade?

### DISCUSSION THREAD

**Discussion by ghost** [DFHack/dfhack#3914] [Q&A] [answered]
Is it save to backup the **dfhack-config** folder and restore its contents after upgrading to a new DFHack version? If not: what are the steps to not to lose any configurations made?

**Comment by lethosor** [comment, marked as answer]
Generally, yes. We generally avoid making breaking changes to config formats, and if we do, they would generally only affect individual config files.

**Reply by ghost** [reply]
Thanks!
